.github/ISSUE_TEMPLATE/release.md

```diff
@@ -10,16 +10,31 @@ assignees: ''
 ## 1. Preparation on Thursday
 
 - [ ] Merge pull requests in the [Ready To Go] column
-- [ ] Include translations: `tx pull --force`
+- [ ] Include translations 
+  <details><summary>Command line instructions:</summary>
+    <pre>
+    git checkout master
+    git pull upstream master
+    tx pull --force
+    git commit -a -m "Update all locales with the latest Transifex translations"
+    git push upstream master</pre>
+  </details>
+- [ ] Create a tag:
+  <details><summary>Command line instructions:</summary>
+    <pre>
+    git tag vX.X.X
+    git push upstream vX.X.X</pre>
```

Comment:
What do you think about a one-liner?
```suggestion
    git push upstream HEAD:refs/tags/vX.X.X</pre>
```

Reply:
👍 Nice one, I guess this won't store the tag locally but we don't need it locally (and can always fetch tags if needed).

Reply:
I couldn't help but go further, to remove the now-unnecessary `<details>` toggle.

Reply:
It may be nice to document how to view current tags easily. I use an alias which is doing this:
```
git tag --list 'v*' --sort v:refname | tail -3
```

Reply:
Nice one. I'm fighting the temptation to implement a script to automate this further.. so if you'd like to add that in (maybe with the <details> toggle) feel free too ;)